src/main.cc

```diff
@@ -100,6 +105,32 @@ int main()
     << "-----" << std::endl << std::endl;
   ecm.Each(printAllUpdatedComponents);
 
+  // remove components from the first entity to see how views are updated
+  std::cout << std::endl << "-----" << std::endl << std::endl
+    << "Removing the following components:" << std::endl
+    << "\t- position component from Entity " << entities[0] << std::endl
+    << "\t- linear velocity component from Entity " << entities[1] << std::endl
+    << "\t- linear acceleration component from Entity " << entities[2]
+    << std::endl;
+  ecm.RemoveComponent<Position>(entities[0]);
+  ecm.RemoveComponent<LinearVelocity>(entities[1]);
+  ecm.RemoveComponent<LinearAcceleration>(entities[2]);
+  std::cout << "Done removing the components" << std::endl << std::endl
+    << "-----" << std::endl << std::endl;
+  std::cout << "The following entities have position, linear velocity, and "
+    << "linear acceleration components:" << std::endl;
+  ecm.Each(printAllComponents);
+
+  // add components back in and check the views again
```

Comment:
I should add a check using a view with just 2 components and maybe create a brand new entity after this view is created and add components to make sure this is added to the view (if so, the final print statement should have 6 views, so update the comment at the end of this `main`)

Reply:
https://github.com/adlarkin/simple_ECM/pull/2/commits/11c211996967482265998706bb6a7231f4071097